rules-emerging-threats/2023/TA/Diamond-Sleet/proc_creation_win_apt_diamond_sleet_indicators.yml

```diff
@@ -0,0 +1,21 @@
+title: Diamond Sleet APT Process Activity Indicators
+id: b5495d8d-24ad-4a44-8caf-ceae9a07a5c2
+status: experimental
+description: Detects process creation activity indicators related to Diamond Sleet APT
+references:
+    - https://www.microsoft.com/en-us/security/blog/2023/10/18/multiple-north-korean-threat-actors-exploiting-the-teamcity-cve-2023-42793-vulnerability/
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/10/24
+tags:
+    - attack.execution
+    - detection.emerging_threats
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains: ' uTYNkfKxHiZrx3KJ'
```

Comment:
```suggestion
        CommandLine|endswith: ' uTYNkfKxHiZrx3KJ'
```

Reply:
Its unique enough that we don't need this endswith